src/main/java/com/epam/prd/jjop/helpers/arrays/Copy.java

```diff
@@ -0,0 +1,184 @@
+package com.epam.prd.jjop.helpers.arrays;
+
+import java.util.Arrays;
+
+/**
+ * Demonstrates possibilities of making copies of array by using static methods copyOf
+ * and copyOfRange from {@link java.util.Arrays} helper class
+ * Compares java.util.Arrays.copyOf vs java.util.Arrays.copyOfRange vs java.lang.System.arraycopy() methods
+ *
+ * @author Svetlana_Zubkova
+ */
+class Copy {
+
+    public static void main(String[] args) {
+
+        // Creates some arrays to copy
+        int[] arraySource = {1, 2, 3, 4, 5, 6};
+        int[] arrayDestination = {0, 0, 0, 0, 0, 0, 0, 0};
+        int[] intsArray = {1, 2, 3, 4, 5, 6, 7, 8};
+        boolean[] booleansArray = {true, false, true, false, false, true};
+        byte[] bytesArray = {4, 1, 8, 2, 2, 3, 16, 4, 4};
+        char[] charsArray = {'a', 'e', 'i', 'o', 'u'};
+        double[] doublesArray = {12, 3, 17, 5, 8, 17, 2, 37};
+        float[] floatsArray = {4.12f, 7.3f, 8.17f, 1.5f, 7.8f, 4.17f, 3.2f, 6.37f};
+        long[] longsArray = {4, 54, 56, 584, 51, 84, 5, 28, 33, 9};
+        short[] shortsArray = {10, 8, 6, 4, 2, 0};
+        String[] stringsArray = new String[]{"apple", "pie"};
+        Number[] numbersArray = new Integer[]{1, 4};
+
+
+        /**
+         * Firstly lets look at  {@link java.lang.System#arraycopy(Object, int, Object, int, int)} method that
+         * Copies an array from the specified source array, beginning at the specified position,
+         * to the specified position of the destination array.
+         * A subsequence of array components are copied from the source array referenced by <b>src/b>
+         * to the destination array referenced by <b>dest</b>.
+         * The number of components copied is equal to the length argument.
+         *
+         * <p>Declaration method:</p>
+         * public static void arraycopy(Object source_arr, int sourcePos, Object dest_arr, int destPos, int len)
+         * @param source_arr - the source array.
+         * @param sourcePos - starting position in the source array from where to copy.
+         * @param dest_arr - array to be copied in.
+         * @param destPos - starting position in the destination data, where to copy in.
+         * @param len -the number of array elements to be copied.
+         */
+
+        /**
+         * Uses method java.lang.System.arraycopy() to copy part of Source array to Destination one.
+         */
```

Comment:
Don't divide a javadoc into two parts, leaving one of those hanging int the air.